Description: fix build with hunspell 1.4
 hunspell 1.4.0 doesn't expose MAXWORDLEN anymore.
Author: Caolan McNamara <caolanm@redhat.com>
Origin: upstream bug
Bug: http://bugzilla.abisource.com/show_bug.cgi?id=13772
Bug-Debian: https://bugs.debian.org/821464

diff -ru enchant-1.6.0-orig/src/myspell/myspell_checker.cpp enchant-1.6.0/src/myspell/myspell_checker.cpp
--- enchant-1.6.0-orig/src/myspell/myspell_checker.cpp	2016-04-18 12:25:00.094614256 +0100
+++ enchant-1.6.0/src/myspell/myspell_checker.cpp	2016-04-18 12:26:09.108569576 +0100
@@ -148,6 +148,10 @@ MySpellChecker::~MySpellChecker()
 		g_iconv_close(m_translate_out);
 }
 
+#ifndef MAXWORDLEN
+#   define MAXWORDLEN 100
+#endif
+
 bool
 MySpellChecker::checkWord(const char *utf8Word, size_t len)
 {
